README.md

```diff
@@ -188,16 +188,20 @@ There are two ways of enabling service access: via the pipeline and manually:
 
 ### Enable via the pipeline (Preferred)
 
-The pipeline uses [https://github.com/cloud-gov/cg-pipeline-tasks/blob/main/set-plan-visibility.yml](https://github.com/cloud-gov/cg-pipeline-tasks/blob/main/set-plan-visibility.yml) to set the list of orgs to enable the broker access.  To add additional organizations, modify the `cg-deploy-autoscaler.yml` secrets file, the list of orgs are space delimited: 
+The pipeline uses [https://github.com/cloud-gov/cg-pipeline-tasks/blob/main/set-plan-visibility.yml](https://github.com/cloud-gov/cg-pipeline-tasks/blob/main/set-plan-visibility.yml) to set the list of orgs to enable the broker access.  There are credhub variables for each of the environments which are space delimited. To  To add additional organizations, modify the following credhub variables: 
 
 ```
-cf:
-  development:
-    service_organization: org1 org2 org3
-  staging:
-    service_organization: org1 org2
-  production:
-    service_organization: org3 org4 org5
+- name: service_organization_development
```

Comment:
```suggestion
- name: service_organization_development
```

I assume the full name of the credhub variable is longer, but perhaps not safe to document here in the repo?